data/json/monsters/mammal.json

```diff
@@ -504,6 +511,7 @@
     "anger_triggers": [ "HURT", "FRIEND_ATTACKED", "FRIEND_DIED" ],
     "placate_triggers": [ "MEAT" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Dog has size "SMALL", why give it harvest animal_medium? It needs to be a new category for animal_small.

Also will need to remove the FUR/BONES/FAT flags like you've done similarly on line 422.

Reply:
For most dog breeds the amount of meat in animal_medium is consistent with the rest of the game. Size should probably be MEDIUM.